### PR TITLE
remove references to cdp

### PR DESCRIPTION
**Pull request by teallarson**
Rephrased to center us as a reverse ETL tool, not a CDP platform.

Not sure how I feel about when to capitalize or not capitalize reverse ETL... open to opinions!

### CODE REVIEW

File: components/solutions/comparisonPage.tsx [hunk 1]
```diff
@@ -75,10 +75,7 @@ export default function ComparisonPage(comp: ComparisonInfo) {
           <br />
         </Container>
       </div>
-      <FeatureBanner
-        category="Customer Data Platform"
-        features={comp.features}
-      />
+      <FeatureBanner category="Data Tool" features={comp.features} />
```
Comment (teallarson) [inline components/solutions/comparisonPage.tsx]: This is used for all three comparisons (Segment, HighTouch, and Census) so it didn't completely make sense to reference Reverse ETL as not all of the competitors are.

File: components/solutions/featureBanner.tsx [hunk 2]
```diff
@@ -9,7 +9,7 @@ export interface FeatureArray extends Array<Feature> {}
 
 export default function FeatureBanner({
   features,
-  category = "Customer Data Platform",
+  category = "Data Tool",
```
Comment (teallarson) [inline components/solutions/featureBanner.tsx]: Used for multiple comparison pages (Segment, Census, High Touch) so didn't refer to reverse ETL here either.

File: components/solutions/comparisonPage.tsx (hunk 1 above)
Comment (andyjih) [reply components/solutions/comparisonPage.tsx]: Hm, i think I'm ok with using "Reverse ETL" in this case to get the words on the page. Segment technically does reverse ETL in addition to various other things, so it's not wrong.

File: components/solutions/featureBanner.tsx (hunk 2 above)
Comment (andyjih) [reply components/solutions/featureBanner.tsx]: Same comment as above. "Reverse ETL" works just fine in this case imo.

File: components/solutions/comparisonPage.tsx (hunk 1 above)
Comment (teallarson) [reply components/solutions/comparisonPage.tsx]: Oh great! Will make that change.

File: pages/blog/google-sheets-source.mdx [hunk 3]
```diff
@@ -2,12 +2,12 @@
 title: "Google Sheets Source"
 date: "2020-04-15"
 author: "Brian Leonard"
-pullQuote: "Grouparoo has the ability to connect to a Google Sheet to bring data into its customer data platform (CDP)."
+pullQuote: "Grouparoo has the ability to connect to a Google Sheet to bring data into its data warehouse."
```
Comment (andyjih) [inline pages/blog/google-sheets-source.mdx]:
```suggestion
pullQuote: "Grouparoo is the Reverse ETL platform to connect Google Sheets data to your SaaS tools"
```

File: pages/blog/google-sheets-source.mdx [hunk 4]
```diff
@@ -2,12 +2,12 @@
...
 image: "google-sheets-source/source-data-preview.png"
 tags: [connections, engineering]
 ---
 
-Grouparoo has the ability to connect to a Google Sheet to bring data into its customer data platform (CDP). This enables all of those crazy sheets out there to be the source of truth for your profiles and be fed into your marketing tools. Don't forget: with great power comes great responsibility!
+Grouparoo has the ability to connect to a Google Sheet to bring data into its data warehouse. This enables all of those crazy sheets out there to be the source of truth for your profiles and be fed into your marketing tools. Don't forget: with great power comes great responsibility!
```
Comment (andyjih) [inline pages/blog/google-sheets-source.mdx]:
```suggestion
Grouparoo is the Reverse ETL platform to connect Google Sheets data to your SaaS tools. This enables all of those crazy sheets out there to be the source of truth for your profiles and be fed into your marketing tools. Don't forget: with great power comes great responsibility!
```

**Review by andyjih — CHANGES_REQUESTED** [review]
A couple copy tweaks. Thanks for tackling this!

File: components/solutions/comparisonPage.tsx [hunk 5]
```diff
@@ -75,10 +75,7 @@ export default function ComparisonPage(comp: ComparisonInfo) {
           <br />
         </Container>
       </div>
-      <FeatureBanner
-        category="Customer Data Platform"
-        features={comp.features}
-      />
+      <FeatureBanner category="Reversel ETL Tool" features={comp.features} />
```
Comment (andyjih) [inline components/solutions/comparisonPage.tsx]:
```suggestion
      <FeatureBanner category="Reverse ETL Tool" features={comp.features} />
```

**Review by andyjih — APPROVED** [review]

**Pull request merged**